posts/git-cherry-pick/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Cherry-Pick on GitHub"
+author: "Robyn Marowitz"
+date: "2023-09-26"
+description: "Committing a mistake in GitHub can feel so scary, cherry picking is a tool to help you fix your history when something like that happens. This post lays out simple and clear instructions on the cherry picking method."
+categories: [github, open-science]
+image: "cherry_pick.jpeg"
```

Comment:
Do you know if this image is CC0 or otherwise licensed without an attribution requirement? Do we need to credit the owner of the image?

Reply:
It's an adobe stock image so I think it's okay to use? https://stock.adobe.com/search?k=cherry+picking